gematria/testing/python/ir_utils_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import textwrap
+import os
+
+from absl.testing import absltest
+from gematria.testing.python import ir_utils
+from pyarrow import parquet
+
+
+class IRUtilsTests(absltest.TestCase):
+
+  def test_get_bc_from_ir(self):
+    ir_string = textwrap.dedent("""\
+    define i32 @a() {
+      ret i32 0
+    }
+    """)
```

Comment:
Nitpicky: Now that we can indent arbitrarily, I'd indent by four more columns (the same way any other arguments of the function would be indented).

Same for the other strings.